rich/logging.py

```diff
@@ -50,6 +59,10 @@ class RichHandler(Handler):
     ]
     HIGHLIGHTER_CLASS: ClassVar[Type[Highlighter]] = ReprHighlighter
 
+    levels: ClassVar[Dict[int, str]] = {}
+    level_style: ClassVar[Dict[int, str]] = {}
```

Comment:
One can also give styles for the names. By default, the handler will use the ones defined in `_level_styles`.